configs/bedrock.conf

```diff
@@ -0,0 +1,28 @@
+# Bedrock default config file.
+# Run `bedrock -?` for help with command line arguments
+
+# Path of db file; will be created if absent
+BEDROCK_DB_PATH="/var/lib/bedrock/bedrock.db"
+
+# If running a cluster, these values need to be different for each node.
+THISNODE="bedrock"
+PRIORITY=200
+
+# Cache size in kilobytes.  Default: 1GB
+CACHE_SIZE="-cacheSize 10000"
+
+# Bind to network address
+SERVER_HOST="0.0.0.0:8888"
+NODE_HOST="0.0.0.0:8889"
+
+# Number of worker threads.  Default: # of CPUs on the system.
+WORKER_THREADS="-workerThreads 1"
```

Comment:
we should leave this commented out, and let bedrock detect and default to the number of multithreaded available cpus.